codes/quantum/qubits/two_dimensional_hyperbolic_surface.yml

```diff
@@ -0,0 +1,106 @@
+
+protection: 'Protects against Pauli errors with distance \( d \propto \log(n) \). Approximate code parameters are \( [[n, (1-2/r - 2/s)  n + 2, \log(n) ]] \)'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: '\( k/n = (1-2/r - 2/s) + 2/n \) which approaches a constant value as the number of physical qubits grows. The distance of the code scales logarithmically with \( n \) and the weight of the stabilizers is \( r \) for \(r\)-checks and \(s\) for \(X\)-checks, ( Recall \( {r,s} \) are the tiling parameters for the hjyperbolic surface)'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders:
+    - 'As with other surface codes, after the surface is tessellated, the physical qubits sit on the edges of the tiles and the logical qubits sit at the center of the unit.'
+    - 'For each family of codes defined by a given tessellation (ie, {r,s}) the encoding rate decreases with $n$, thus a [[n,k,d]] code derived from a {r,s} tiling will have the rate $k/n = 1 - 2/r - 2/s + 2/n$'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+
+
+  decoders:
+    - 'Many decoders developed fro the toric code (surface code) can also be used for hyperbolic surface codes. For example, minimum weight-perfect matching \cite{arXiv:1506.04029} and union find decoder \cite{1709.06218} are both valid decoders for two-dimensional hyperbolic surface codes.'
```

Comment:
This might not be specific enough to be worth keeping. However, if you've found any, please mention any quantitative criteria that determine whether a given decoder is applicable to hyperbolic geometries. Otherwise, please omit.